src/main/kotlin/io/github/facilityapi/intellij/lexer/_FsdLexer.flex

```diff
@@ -88,6 +88,7 @@ MARKDOWN_TEXT=.+
   "errors"                       { yybegin(LIST_BODY); return ERRORS; }
   "enum"                         { yybegin(LIST_BODY); return ENUM; }
 
+  ";"                            { return SEMI; }
```

Comment:
This still doesn't account for markdown after the service body when braces are not used. This is tricky since rather than looking for both service member items and markdown items simultaneously, we look for only service member items then only markdown items when the service body is finished. The lexer works when curly braces are used because it knows when the service body is finished when it sees the right curly brace. This has been nuanced in the other parsers as well.